src/smexperiments/trial_component.py

```diff
@@ -110,8 +110,17 @@ def create(cls, trial_component_name, display_name=None, sagemaker_boto_client=N
             sagemaker_boto_client=sagemaker_boto_client)
 
     @classmethod
-    def list(cls, source_arn=None, created_before=None, created_after=None,
-             sort_by=None, sort_order=None, sagemaker_boto_client=None):
+    def list(
+            cls,
+            source_arn=None,
+            created_before=None,
+            created_after=None,
+            sort_by=None,
+            sort_order=None,
+            sagemaker_boto_client=None,
+            trial_name=None,
```

Comment:
We should support experiment name as well.

Reply:
added